modules/ublog/src/main/UblogApi.scala

```diff
@@ -30,26 +30,28 @@ final class UblogApi(
 
   def getByPrismicId(id: String): Fu[Option[UblogPost]] = colls.post.one[UblogPost]($doc("prismicId" -> id))
 
-  def update(data: UblogForm.UblogPostData, prev: UblogPost)(using me: Me): Fu[UblogPost] =
-    getUserBlog(me.value, insertMissing = true).flatMap { blog =>
+  def update(author: User, data: UblogForm.UblogPostData, prev: UblogPost)(using me: Me): Fu[UblogPost] =
+    getUserBlog(author, insertMissing = true).flatMap { blog =>
       val post = data.update(me.value, prev)
```

Comment:
It's important to keep `me.value` used here, that way we can know which user updated or published the blog of lichess or someone else.